doc/source/reconstruction.rst

```diff
@@ -1,6 +1,46 @@
 Reconstruction
 ==============
 
+Algorithms to recosntruct a graph from time series data.
+
+
+Base class
+----------
+.. autoclass:: netrd.reconstruction.BaseReconstructor
+
+
+Available algorithms
+--------------------
+
+All of the following algorithms inherit from ``BaseReconstructor`` and have
+the same general usage as above.
+
+.. autosummary::
+   :nosignatures:
+
+    netrd.reconstruction.ConvergentCrossMappingReconstructor
+    netrd.reconstruction.CorrelationMatrixReconstructor
+    netrd.reconstruction.CorrelationSpanningTree
+    netrd.reconstruction.ExactMeanFieldReconstructor
```

Comment:
ExactMeanField and NaiveMeanField should be removed and replaced with MeanField.